### DISCUSSION TITLE
Componentes em desenvolvimento

### DISCUSSION THREAD

**Discussion by PMLF** [As-Raparigas-do-Codigo/ardc-website#3] [General]
Thread para marcar as componentes em que pegamos e indicar a branch de cada

**Comment by PMLF** [comment]
Tertiary buttons (branch `tertiary-btn`)

![tertiarybtn](https://user-images.githubusercontent.com/44099019/167260891-96ed7ca9-11e3-43af-a287-408dda63e34b.png)

**Comment by PMLF** [comment]
Sponsors section (branch `sponsors-section`)

![image](https://user-images.githubusercontent.com/44099019/168426444-689ae719-cfb1-498d-9c0b-0254dcea337b.png)

**Comment by MonicaFidalgo** [comment]
Card component -> Talks and Eventos e Artigos recentes (with data files)

(branch `feature/home-talks-and-events`)

<img width="862" alt="Captura de ecrã 2022-05-17, às 22 41 27" src="https://user-images.githubusercontent.com/41929489/168914818-2b25a5c9-3af7-4706-9d64-2faef8f5c202.png">
)

**Comment by MonicaFidalgo** [comment]
Horizontal card -> Home workshops section
(branch feature/home-workshops)
![Captura de ecrã 2022-05-19, às 18 28 50](https://user-images.githubusercontent.com/41929489/169361420-a4f4f078-00ed-4461-bc59-4657cdf27558.png)

**Comment by toothlesspanda** [comment]
Home / Mentores Section 
branch -> feature/home-mentores-section
![image](https://user-images.githubusercontent.com/6379428/169529014-4b0b2025-cdcc-480b-bee1-dc0da1d99a33.png)

**Comment by MonicaFidalgo** [comment]
refactor pages and components
pr: https://github.com/As-Raparigas-do-Codigo/ardc-website/pull/11

**Comment by toothlesspanda** [comment]
Footer
![image](https://user-images.githubusercontent.com/6379428/170090504-f5055ab1-fc2d-477e-8f03-5abb8ae2f2fa.png)

**Comment by inesmartins** [comment]
Como parte [deste](https://github.com/As-Raparigas-do-Codigo/ardc-website/pull/16) pr desenvolvi os componentes numerados, que tanto servem para a página de "colaborar" como de "faq":

![Screenshot 2022-05-28 at 18 42 11](https://user-images.githubusercontent.com/39055313/170836855-438c7641-ce41-45ef-84cc-55c35e725958.png)
